src/System.ComponentModel.Composition/tests/System/ComponentModel/IO/FileIO.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
these files in /IO/*cs duplicate the existing test helpers for creating and using temp files/directories.
leave as is for now but in future someone could eliminate the duplication

Reply:
You have checked in 2 copies of them all though. You only need 1 copy. Please check for dupe C# files